webui/react/src/pages/ExperimentDetails/ExperimentVisualization.module.scss

```diff
@@ -15,5 +15,12 @@
   }
 }
 .alert {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
   padding: var(--theme-sizes-layout-big);
+
+  & > *:nth-child(2) {
```

Comment:
this seems a little vague for a rule, i.e. I'd rather have a comment what part of the built-in Alert component is affected, or a css class to avoid it affecting other elements.

Reply:
good call, I've updated to take a specific named css instead.